### DISCUSSION TITLE
Content duplication when deployed and continued deployment actions

### DISCUSSION THREAD

**Discussion by kcanada** [alshedivat/al-folio#256] [Q&A]
I am unsure how to correct the following issue(s). I have gotten the site to display on my GitHub site. However, when I push changes the deployment fails (many times for many actions) and when I manually deploy from the root folder duplicated pages. I am new to Jekyll deployment, buy could not find much information about this error (online or on Gitter). 

Any help would be appreciated!
![Screen Shot 2021-04-30 at 1 10 28 AM](https://user-images.githubusercontent.com/16561212/116707735-02fc9780-a99d-11eb-8c26-cdf6b884598f.png)
![Screen Shot 2021-04-30 at 1 10 24 AM](https://user-images.githubusercontent.com/16561212/116707795-114ab380-a99d-11eb-9b8e-ebd73a074c27.png)
![Screen Shot 2021-04-30 at 10 17 59 AM](https://user-images.githubusercontent.com/16561212/116708090-5e2e8a00-a99d-11eb-8d3e-7448168ba08e.png)

**Comment by alshedivat** [comment]
This is very odd. Do the files get duplicated automatically after running the deploy script? Could you provide more details?
